plugins/otter-pro/inc/plugins/class-block-conditions.php

```diff
@@ -566,6 +596,122 @@ public function has_products( $products ) {
 		return $bought;
 	}
 
+	/**
+	 * Check based on WooCommerce product category.
+	 *
+	 * @param array $categories IDs of Categories.
+	 *
+	 * @since  2.7.0
+	 * @access public
+	 */
+	public function has_product_category( $categories ) {
+		$in_category = false;
+
+		global $product;
+
+		if ( ! $product ) {
+			return false;
+		}
+
+		$terms = get_the_terms( $product->get_id(), 'product_cat' );
+
+		if ( gettype( $terms ) !== 'array' ) {
```

Comment:
Is there a reason to use `gettype` instead of `is_array`?

Reply:
It was built over an existing method so we I copied that and started to modify. I don't remember why we used `gettype` instead of `is_array` then but I agree it's redundant over `is_array`.